sdk/identity/azure-identity/pom.xml

```diff
@@ -41,6 +41,12 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>msal4j</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>net.minidev</groupId>
+          <artifactId>json-smart</artifactId>
```

Comment:
This exclusion can result in customers running into dependency compatibility issues with json-smart version:

1. Exclusion will prevent maven from determining which json-smart version is compatible with oauth2-oidc-sdk in msal4j and by default follow the top level json-smart version that can be incompatible with oauth2-oidc-sdk.

2. Retaining the json-smart dependency in msal4j will let maven determine the compatible version with it automatically.

We should not exclude this dependency.